releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -2070,8 +2281,8 @@
   },
   "124419": {
     "commit": "ba05a8deb3cfcc52a9f80bf4e2303cd552fc582b",
-    "text": "kubeadm: mark the sub-phase of 'init kubelet-finilize' called 'experimental-cert-rotation' as deprecated and print a warning if it is used directly; it will be removed in a future release. Add a replacement sub-phase 'enable-client-cert-rotation'.",
-    "markdown": "Kubeadm: mark the sub-phase of 'init kubelet-finilize' called 'experimental-cert-rotation' as deprecated and print a warning if it is used directly; it will be removed in a future release. Add a replacement sub-phase 'enable-client-cert-rotation'. ([#124419](https://github.com/kubernetes/kubernetes/pull/124419), [@neolit123](https://github.com/neolit123)) [SIG Cluster Lifecycle]",
+    "text": "'kubeadm: marked the sub-phase of ''init kubelet-finilize'' called ''experimental-cert-rotation''\nas deprecated and print a warning if it is used directly; it will be removed in\na future release. Add a replacement sub-phase ''enable-client-cert-rotation''.'",
+    "markdown": "'kubeadm: marked the sub-phase of ''init kubelet-finilize'' called ''experimental-cert-rotation''\n  as deprecated and print a warning if it is used directly; it will be removed in\n  a future release. Add a replacement sub-phase ''enable-client-cert-rotation''.' ([#124419](https://github.com/kubernetes/kubernetes/pull/124419), [@neolit123](https://github.com/neolit123))",
```

Comment:
```suggestion
    "markdown": "'kubeadm: marked the sub-phase of ''init kubelet-finilize'' called ''experimental-cert-rotation''\n  as deprecated and print a warning if it is used directly; it will be removed in\n  a future release. Add a replacement sub-phase ''enable-client-cert-rotation''.' ([#124419](https://github.com/kubernetes/kubernetes/pull/124419), [@neolit123](https://github.com/neolit123)) [SIG Cluster Lifecycle]",
```